packages/python-mmh3/PKGBUILD

```diff
@@ -0,0 +1,37 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgbase=python-mmh3
+pkgname=('python-mmh3')
+_pkgname=mmh3
+pkgver=2.5.1
+pkgrel=1
+pkgdesc='Python wrapper for MurmurHash (MurmurHash3), a set of fast and robust hash functions'
+arch=('any')
+url='https://pypi.org/project/mmh3/#files'
+license=('Creative')
+depends=('python')
+makedepends=('python-setuptools')
+source=("https://files.pythonhosted.org/packages/fa/7e/3ddcab0a9fcea034212c02eb411433db9330e34d626360b97333368b4052/$_pkgname-$pkgver.tar.gz")
+sha1sums=('abe5fd3f06322fa814961fa9466ee089fa561a8a')
+sha256sums=('185209a217c52afe43e079e5b232d0ef0f3a262601eaaf4371326ab6dcbec508')
+sha512sums=('1cc2bdaceda00e90355fcca990517aec77e60c2117e716cf7311a5b4aa1a78635cee16ad4d551d73ea72a18fe9c40fb24b7ff042171eae252cc4fb5660592d37')
```

Comment:
Keeps only sha512sums it's not useful to have all of them also I wonder if it will mess up with updpkgsums.